rest-service/manager_rest/test/endpoints/test_tenants.py

```diff
@@ -0,0 +1,85 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+from mock import patch
+from manager_rest import constants, config
+from manager_rest.test import base_test
+from manager_rest.storage import models
+from manager_rest.test.attribute import attr
+
+from cloudify_rest_client.exceptions import CloudifyClientError
+
+CREDENTIALS_PERMISSION = 'tenant_rabbitmq_credentials'
+
+
+@attr(client_min_version=3, client_max_version=base_test.LATEST_API_VERSION)
+@patch('manager_rest.security.secured_resource.premium_enabled', False)
+class TenantsTestCase(base_test.BaseServerTestCase):
```

Comment:
I will change the class name to include community, so it will be more obvious on test report and you could remove the comment below